PurchasesHybridCommon.podspec

```diff
@@ -0,0 +1,36 @@
+Pod::Spec.new do |s|
+  s.name             = "PurchasesHybridCommon"
+  s.version          = "1.0.14"
+  s.summary          = "Common files for hybrid SDKs for RevenueCat's Subscription and in-app-purchase backend service."
+
+  s.description      = <<-DESC
+                       Save yourself the hastle of implementing a subscriptions backend. Use RevenueCat instead https://www.revenuecat.com/
+                       DESC
+
+  s.homepage         = "https://www.revenuecat.com/"
+  s.license          =  { :type => 'MIT' }
+  s.author           = { "RevenueCat, Inc." => "support@revenuecat.com" }
+  s.source           = { :git => "https://github.com/revenuecat/purchases-hybrid-common.git", :tag => s.version.to_s }
+  s.documentation_url = "https://docs.revenuecat.com/"
+
+  s.framework      = 'StoreKit'
+
+  s.ios.deployment_target = '9.0'
+  s.osx.deployment_target = '10.12'
+  s.watchos.deployment_target = '6.2'
+  s.tvos.deployment_target = '9.0'
+
+  s.pod_target_xcconfig = { 'DEFINES_MODULE' => 'YES' }
+
+  s.source_files = [
+    'ios/PurchasesHybridCommon/PurchasesHybridCommon/PurchasesHybridCommon.h',
+    'ios/PurchasesHybridCommon/PurchasesHybridCommon/**/*.h',
+    'ios/PurchasesHybridCommon/PurchasesHybridCommon/**/*.m'
+  ]
+
+
+  s.public_header_files = [
+    'ios/PurchasesHybridCommon/PurchasesHybridCommon/Public/*.h'
+  ]
+
+end
```

Comment:
Next step would be to remove the submodule and add the dependency here right?

Reply:
yep, will do it as a follow-up PR